grafana-plugin/src/containers/Alerts/Alerts.tsx

```diff
@@ -164,4 +190,24 @@ export const Alerts = function () {
         !getItem(AlertID.CONNECTIVITY_WARNING)
     );
   }
-};
+
+  /**
+   * tbh we don't really need the `currentUserPk` reference here...
+   * the only reason why it's here is to appease mobx. Without this reference, the `@computed` property
+   * on `UserStore.currentUser` doesn't recalculate and will just be stuck on returning `undefined`..
+   *
+   * If we dereference `currentUserPk` here, even if we don't use it.. things just seem to work
+   * (what is this `mobx` wizardry?)
+   *
+   * Seems to be related to this https://stackoverflow.com/questions/77724466/mobx-computed-not-updating
+   */
+  function showCurrentUserGoogleOAuth2TokenIsMissingScopes(): boolean {
+    return Boolean(
+      currentUserPk &&
+        currentUser &&
+        currentUser.has_google_oauth2_connected &&
+        currentUser.google_oauth2_token_is_missing_scopes &&
+        !getItem(AlertID.USER_GOOGLE_OAUTH2_TOKEN_MISSING_SCOPES)
+    );
+  }
```

Comment:
@grafana/grafana-oncall-frontend for the life of me I can't figure this out 😆

it works, but if I delete the reference to `currentUserPk` (which isn't actually needed), it stops working